CMakeLists.txt

```diff
@@ -62,6 +65,17 @@ add_cts_option(SYCL_CTS_ENABLE_OPENCL_INTEROP_TESTS
 
 add_cts_option(SYCL_CTS_ENABLE_CUDA_INTEROP_TESTS
     "Enable CUDA interoperability tests", OFF)
+
+# ------------------
+if(SYCL_CTS_ENABLE_OPENCL_INTEROP_TESTS OR SYCL_IMPLEMENTATION STREQUAL "ComputeCpp")
```

Comment:
Could you add a comment here with the clarification why ComputeCPP case is special?